### PR TITLE
Add missing links to Rc doc

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
r? @frewsxcv

### CODE REVIEW

File: src/liballoc/rc.rs [hunk 1]
```diff
@@ -251,9 +251,11 @@ struct RcBox<T: ?Sized> {
 /// See the [module-level documentation](./index.html) for more details.
 ///
 /// The inherent methods of `Rc` are all associated functions, which means
-/// that you have to call them as e.g. `Rc::get_mut(&value)` instead of
-/// `value.get_mut()`.  This avoids conflicts with methods of the inner
+/// that you have to call them as e.g. [`Rc::get_mut(&value)`][get_mut] instead of
```
Comment (frewsxcv) [inline src/liballoc/rc.rs]: it looks like the rest of the file uses 79 (or 80?) characters for line length. Can you make this line match?

File: src/liballoc/rc.rs [hunk 2]
```diff
@@ -127,20 +114,20 @@
 //! ```
 //!
 //! If our requirements change, and we also need to be able to traverse from
-//! `Owner` to `Gadget`, we will run into problems. An `Rc` pointer from `Owner`
+//! `Owner` to `Gadget`, we will run into problems. An [`Rc`] pointer from `Owner`
```
Comment (frewsxcv) [inline src/liballoc/rc.rs]: Same comment as https://github.com/rust-lang/rust/pull/38189/files#r91204663

**Comment by frewsxcv** [comment]
r=me after line lengths are fixed

File: src/liballoc/rc.rs (hunk 1 above)
Comment (GuillaumeGomez) [inline src/liballoc/rc.rs]: In a lot of places, it goes beyond 80 columns so I don't see it as an issue?

File: src/liballoc/rc.rs (hunk 2 above)
Comment (GuillaumeGomez) [inline src/liballoc/rc.rs]: And same answer. :p

**Comment by frewsxcv** [comment]
You're right, looks like it's 100 in other places in this file.

@bors r+ rollup

**Pull request merged**